design-documents/static-checks-for-templates.md

```diff
@@ -0,0 +1,27 @@
+### Overview
+
+At the time of this writing only 85% of the `*.phtml` files in Magento core are being statically 
+tested due to `@codingStandardsIgnore` present in over 700 templates. There is a current security initiative 
+that requires a large number of templates to be modified and they need to be covered by static checks moving 
```

Comment:
https://jira.corp.magento.com/browse/MC-15343

Reply:
I don't think maintainers have access to JIRA after acquisition by Adobe anymore.

Is it too sensitive for public discussion or at least in some private Slack channel?

Reply:
It's sort of sensitive. The changes will go to `2.3-develop`. The changes essentially involve cleaning certain things up in every module.

Reply:
Ok. I believe it would be enough to share what kind of changes to code and tests is supposed to be done so that it can be aligned with other coding standard compliance activities.

Reply:
In scope of this task the changes are described in this proposal - cleanup templates (by removing phpcs suppression).